.github/workflows/verify.yml

```diff
@@ -0,0 +1,74 @@
+name: Verify Submodule Pull Request
```

Comment:
Can you point me to a test run where the verification properly failed?

Reply:
I've linked some in the issue description in Jira.

Reply:
Here are some new ones:
- [Ensure Current PR Title Includes Issue Code](https://github.com/Lombiq/Orchard-Data-Tables/actions/runs/2458383201)
- [Ensure Parent Repository Contains Matching PR](https://github.com/Lombiq/Orchard-Data-Tables/runs/6785308647?check_suite_focus=true#step:5:13)

Reply:
Thanks.